suite-native/intl/src/en.ts

```diff
@@ -247,18 +247,27 @@ export const en = {
         },
     },
     moduleDevice: {
-        IncompatibleDeviceModalAppendix: {
+        incompatibleFirmwareModalAppendix: {
```

Comment:
Why capitalize first letter? I'd prefer to keep it consistent with camelcase, unless we want to change it everywhere?

Reply:
It was PascalCase and I made it camelCase...

Reply:
Hah, sorry 😄 I guess my brain was still half asleep and switched the colors in my head 🤔